### PR TITLE
ws: Add 'LoginTo' cockpit.conf option

### PR DESCRIPTION
**Pull request by stefwalter**
This option configures whether the 'Connect To' option on
the login page is visible or not.

In addition if the option is not specified, the default comes
from whether cockpit-ssh is present and executable.

### CODE REVIEW

File: src/ws/cockpithandlers.c [hunk 1]
```diff
@@ -241,6 +241,30 @@ add_oauth_to_environment (JsonObject *environment)
...
+  JsonObject *page;
+  const gchar *value;
+
+  page = json_object_new ();
+
+  value = cockpit_conf_string ("WebService", "LoginTitle");
+  if (value)
+    json_object_set_string_member (page, "title", value);
+
+  if (page_login_to < 0)
```
Comment (petervo) [inline src/ws/cockpithandlers.c]: This if seems unnecessary, page_login_to is always -1.

File: test/verify/check-login [hunk 2]
```diff
@@ -99,6 +99,29 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
...
+
+        # Default options be to display these options
+        m.execute("rm /etc/cockpit/cockpit.conf")
+        m.restart_cockpit()
+        b.open("/system")
+        b.wait_present("#option-group")
+        b.wait_visible("#option-group")
+
+        # And now we remove cockpit-ssh which affects the default
+        m.execute("rm -f /usr/libexec/cockpit-ssh /usr/lib/cockpit/cockpit-ssh")
```
Comment (petervo) [inline test/verify/check-login]: This doesn't work for atomic, where cockpit-ssh is inside the container.

File: src/ws/cockpithandlers.c (hunk 1 above)
Comment (stefwalter) [inline src/ws/cockpithandlers.c]: It's a static variable. It keeps the last value set.

File: test/verify/check-login (hunk 2 above)
Comment (stefwalter) [inline test/verify/check-login]: Ah, good point. Will fix.

**Pull request closed without merging**